pkgs/applications/audio/stargate/default.nix

```diff
@@ -0,0 +1,169 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, substituteAll
+, alsa-lib
+, autoconf
+, automake
+, ffmpegSupport ? true
+, ffmpeg
+, fftw
+, fftwSinglePrec
+, file
+, gdb
+, gettext
+, git
+, jq
+, lameSupport ? true
+, lame
+, libsbsms
+, libsndfile
+, pkg-config
+, portaudio
+, portmidi
+, python3
+, python3Packages
+, rubberband
+, stargate-libcds
+, vorbisToolsSupport ? true
+, vorbis-tools
+}:
+
+assert ffmpegSupport -> ffmpeg != null;
+assert lameSupport -> lame != null;
+assert vorbisToolsSupport -> vorbis-tools != null;
+
+let
+  python3CommonInputs = python3.withPackages
+    (ps: with ps; [
+      jinja2
+      mido
+      mutagen
+      numpy
+      psutil
+      pymarshal
+      pyqt5
+      pyyaml
+      wavefile
+      wheel
+    ]);
+in
+stdenv.mkDerivation rec {
+  pname = "stargate";
+  version = "21.11.5";
+  disabled = python3Packages.pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "stargateaudio";
+    repo = pname;
+    rev = "release-${version}";
+    sha256 = "sha256-d5ZuDCyq9gvfZvq57XHKEq0pvi+dWgz6z34kFIOmj7A=";
+  };
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    gdb
+    git
```

Comment:
How is git used in the build process? If it is grabbing the version it will probably not work and we would need to patch it.

Reply:
```
commit_hash:
	git rev-parse --short HEAD > COMMIT || echo unknown > COMMIT
```

i just removed the git dependency